app/assets/stylesheets/govuk_publishing_components/components/govspeak/_attachment.scss

```diff
@@ -9,11 +9,6 @@
 
 .govspeak, // Legacy class name that's still used in some content items - needs to be kept until `.govspeak` is removed from the content items.
 .gem-c-govspeak {
-  // Scope attachment and attachment-link component styles to gem-c-govspeak
-  @import "../attachment-link";
-  @import "../attachment";
-  @import "../details";
```

Comment:
Moving the details component stylesheet to the govspeak view template does not appear to cause any rendering issues when tested on https://www.gov.uk/government/groups/secretary-of-state-for-transports-honorary-medical-advisory-panel-on-driving-and-psychiatric-disorders